pages/rooms/create/mood/index.tsx

```diff
@@ -27,9 +29,40 @@ const MOOD_EXAMPLE = [
 ];
 
 const RoomCreateMoodPage: NextPage = () => {
-  const [mood, setMood] = useState("");
+  const [mood, setMood] = useState(
+    {} as {
+      name: string;
+      emoji: string;
+    }
+  );
   const router = useRouter();
 
+  const createRoom = () => {
+    const { roomName } = router.query as { roomName: string };
+
+    postRoom({
+      description: roomName,
+      moods: [
+        {
+          name: mood.name,
+          emoji: mood.emoji,
+        },
+      ],
+    })
+      .then((room: Room) => {
```

Comment:
postRoom이 잘 타이핑 되어있다면 여기는 타입 추론을 이용하면 좋을 거 같아
```suggestion
      .then((room) => {
```

Reply:
[82332ec](https://github.com/mash-up-kr/musily-web/pull/85/commits/82332ec8a1aad4f93ecf4dfd2eda96aec53ada29) 커밋에서 수정했어요~